std/StringTools.hx

```diff
@@ -413,6 +413,15 @@ class StringTools {
 		return false;
 		#end
 	}
+	
+	/**
+		Converts the first character of the String `s` to upper case and the 
+		remaining characters to lower case.
```

Comment:
...as well as in [Ruby](http://ruby-doc.org/core-2.2.0/String.html#method-i-capitalize)

This method actually comes in very handy when doing lots of dynamic text formatting (e.g. using specific words at the beginning of a sentence instead of somewhere in the middle.).

How about we add a space_delim:Bool that makes sure anything after the first white space in the string, if encountered,  stays intact?